src/notation/internal/notationuiactions.cpp

```diff
@@ -462,11 +462,23 @@ const UiActionList NotationUiActions::m_actions = {
              TranslatableString("action", "Toggle rhythmic sl&ash notation"),
              TranslatableString("action", "Toggle rhythmic slash notation")
              ),
+    UiAction("enh-both",
+             mu::context::UiCtxProjectOpened,
+             mu::context::CTX_NOTATION_OPENED,
+             TranslatableString("action", "Change enharmonic spelling (concert and transposed &pitch)"),
+             TranslatableString("action", "Change enharmonic spelling (concert and transposed pitch)")
+             ),
+    UiAction("enh-current",
+             mu::context::UiCtxProjectOpened,
+             mu::context::CTX_NOTATION_OPENED,
+             TranslatableString("action", "Change enharmonic spelling (&current pitch mode only)"),
+             TranslatableString("action", "Change enharmonic spelling (current pitch mode only)")
+             ),
```

Comment:
I'm confused, why are these 2 intries in this file/list twice, here and further down?

Reply:
It was... seems I added stufff that was already there instead if modifying the latter
May explain wha the accelerator keys didn't show...